src/app/pages/deployment-network/deployment-network.component.html

```diff
@@ -46,8 +46,8 @@
   </div>
 </div>
 <div *ngIf="compareDeployedNetwork()" class="ui-messages ui-messages-success d-block" role="alert">
-  You are on <b>{{ this.web3Service.network }} </b> and are able to use this DApp!
+  You are connected to  '<b>{{ this.web3Service.network }} </b>' ,You can use our DApp!
 </div>
 <div *ngIf="!compareDeployedNetwork()" class="ui-messages ui-messages-danger d-block" role="alert">
-  You are on <b>{{ this.web3Service.network }} </b> and are not able to use this DApp!
-</div>
\ No newline at end of file
+  You are not able to use this DApp, because you are connected to '<b>{{ this.web3Service.network }} </b>'.
```

Comment:
If the user does not have an injected Web3 (this.web3Service.network is empty) the message showed to be something like: Use an Ethereum Wallet extension, or an Ethereum DApp-compatible browser, to be able to use this DApp.